db/migrate/20250304074934_backfill_api_key_scope_modes.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
```

Comment:
Just realised that we should probably do this in a post deploy migration as there is a possibility that an API key is created after this migration runs but before the old app code has been swapped out during the deploy process. 

Maybe a migration to mark the column as `null: false` first and a post-deploy migration to backfill the scopes?

Reply:
I will switch this back to a post-migration. 👍 

> Maybe a migration to mark the column as `null: false` first and a post-deploy migration to backfill the scopes?

Can't do this, because setting `null: false` with `NULL` values in the DB will raise an exception.